distributed/client.py

```diff
@@ -24,9 +24,10 @@
 from functools import partial
 from numbers import Number
 from queue import Queue as pyQueue
-from typing import ClassVar
+from typing import Awaitable, ClassVar, Sequence
 
 from tlz import first, groupby, keymap, merge, partition_all, valmap
+from typing_extensions import Literal
```

Comment:
This is causing the docs build to fail. I noticed that other such occurrences are gated by a `TYPE_CHECKING` check

https://github.com/dask/distributed/blob/7a3ea4c34ae0af6af1137fe3e3bda927aa9eac26/distributed/protocol/compression.py#L30-L31